pages/docs/environments/environment-variables.mdx

```diff
@@ -92,3 +92,4 @@ export default ({ children }) => (
     {children}
   </DocsLayout>
 );
+```
```

Comment:
This is breaking the build

Reply:
thank you! not sure why that gets added each time 😢

Reply:
I think you might need to add this file to `.prettierignore`.

The support for MDX has some issues, including triple backticks.